SPECS/kernel/config

```diff
@@ -4841,6 +4841,7 @@ CONFIG_INFINIBAND_USER_MEM=y
 CONFIG_INFINIBAND_ON_DEMAND_PAGING=y
 CONFIG_INFINIBAND_ADDR_TRANS=y
 CONFIG_INFINIBAND_ADDR_TRANS_CONFIGFS=y
+CONFIG_INFINIBAND_VIRT_DMA=y
```

Comment:
Are these config changes intended?

Reply:
Yes, these config changes are intended. They are new configs introduced with 5.4.81 and are automatically-selected to their default values. The default value is based on logic of any dependency configs and/or the presence of explicitly defined default values.